test/others/test_utils.py

```diff
@@ -168,6 +168,20 @@ def test_convert_files_to_docs(samples_path):
     assert documents and len(documents) > 0
 
 
+def test_get_filename_extension_from_url_zip():
```

Comment:
Please mark this test as a unit test.
We are reorganizing the tests, so in some cases, the marker is missing but is needed to execute them.

```suggestion
@pytest.mark.unit
def test_get_filename_extension_from_url_zip():
```

Reply:
[ee1f6c8](https://github.com/deepset-ai/haystack/pull/5199/commits/ee1f6c8ed51411a43d6c8dabd66183f8d2eda670) add marker